contents/blog/slack-closure.md

```diff
@@ -0,0 +1,46 @@
+---
+date: 2023-20-12
+title: "Why we're closing our public Slack community"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: CEO diaries
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+---
+
+One of the great things about PostHog being open source is that we’ve grown a vibrant community around the project. Since launch we’ve accepted code from over 500 contributors and swapped ideas with thousands of users in our public Slack group. 
+
+Now though, we’ve decided to close our public Slack group and focus efforts elsewhere. In this post I’ll explain why we’re doing this, and what comes next for the PostHog community. 
+
+> **Tl;dr:** We’re closing the Slack group on XX/XX/XX and inviting members to join us in the new [PostHog Community](/questions) instead. This decision only impacts the public Slack group, not the private Slack channels for users spending over $2,000 per month.
+
+## Why are we closing the PostHog Slack?
+
+PostHog has grown incredibly fast over the past four years and while that has ensured the public Slack group has remained a busy place, we think it’s arguably become _too_ busy. We often find users asking questions that have already been answered in previous chats, or re-asking unanswered questions if they drift too far back in the chat history. 
+
+This is a limitation of Slack itself. Message history disappears too quickly and search engines can’t index the chat, so everything which happens is too quickly lost.
+
+Over time we’ve tried a number of things to try and offset this, such as reorganizing channels or adding integrations to impose order. We even [built our own AI bot](/blog/aruba-hackathon#maxai-our-friendly-posthog-support-ai) which would answer questions automatically!
+
+We’ve also investigated moving to a paid plan, but the cheapest option is $7.25 USD per user per month. That’s a prohibitive cost for a channel with over 5,000 users and climbing, and would only increase the history to 30 days. 
+
+## Where will the community go?
+
+In order to ensure users can get the best possible experience, we’ve built our [own community platform](/questions) directly into PostHog.com. You can ask and answer questions, select preferred solutions, filter by topic, participate in AMAs, and search thoroughly to find the information you need. 
+
+We strongly recommend all users [create a community account](/questions). You can, of course, continue to use the in-app support for reporting bugs, giving feedback, or asking for support. We’ll also continue to operate the dedicated Slack channels for users who spend over $2,000 per month. 
+
+In the future we plan to add even more to the PostHog community, including the ability to vote on questions and unlock unique benefits — but already it offers a better alternative to the public Slack group. More than XXX users have asked over XXX questions in the last six months of us building it!
```

Comment:
'unlock unique benefits' - if we can't be specific, prob leave out?

Reply:
```suggestion
In the future we plan to add even more to the PostHog community, including the ability to vote on questions — but already it offers a better alternative to the public Slack group. More than XXX users have asked over XXX questions in the last six months of us building it!
```